.github/workflows/prefer_typescript.yml

```diff
@@ -6,26 +6,43 @@ on:
       - master
 
 jobs:
-  comment:
-    name: Comment about preferring TypeScript
+  check:
     runs-on: ubuntu-latest
     steps:
       - name: Get changed files
         id: changed
         uses: trilom/file-changes-action@master
         with:
           githubToken: ${{ secrets.GITHUB_TOKEN }}
+
       - name: Determine if a .js or .jsx file was added
         id: check
         run: |
-          echo ::set-output name=was_js_file_added::$(jq 'map(endswith(".js") or endswith(".jsx"))' ${HOME}/files_added.json | jq 'reduce .[] as $is_js (false; . or $is_js)')
-      - if: steps.check.outputs.was_js_file_added != 'false'
-        name: Comment about preferring TypeScript
+          js_files_added() {
+            jq -r '
+              map(
+                select(
+                  (contains("cypress-base/") | not) and
+                  (endswith(".js") or endswith(".jsx"))
+                )
+              ) | join("\n")
+            ' ${HOME}/files_added.json
+          }
+          echo ::set-output name=js_files_added::$(js_files_added)
+
+      - if: steps.check.outputs.js_files_added
```

Comment:
i think this might need to still have the string comparison? how did you test this?

Reply:
oh, i see your link to your fork. nevermind

Reply:
Yeah, fortunately empty string is also considered falsy in GitHub actions.